tests/binaries/host/glibcs/2.42/ld-linux-x86-64.so.2

```diff
@@ -0,0 +1 @@
+ld-2.42.so
```

Comment:
This is a symbolic link, right? GitHub should mark it as such in here too not just in the tab on the left, heh...